platforms/enterprise/enterprise/src/integTest/groovy/org/gradle/internal/enterprise/GradleEnterprisePluginCheckInFixture.groovy

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2020 the original author or authors.
+ * Copyright 2023 the original author or authors.
```

Comment:
Shouldn't it be 2024?

Reply:
This was not intended to change. Reverted in https://github.com/gradle/gradle/pull/28673/commits/173282a6df614b11c141ed238c06a1401bd40a3c